### PR TITLE
Hotfix 1.2.9

### PR DESCRIPTION
**Pull request by jenniferboedker**
1.2.9 (2021-12-20)
----------------------------------------------

**Added**

**Fixed**

* Fix CVE-2021-45105

**Dependencies**

* org.apache.logging.log4j:log4j-api: 2.16.0 -> 2.17.0
* org.apache.logging.log4j:log4j-core: 2.16.0 -> 2.17.0

### CODE REVIEW

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Very nice! 
I'll approve the PR once you also updated the Version indicated in the [life.qbic.AppInfo ](https://github.com/qbicsoftware/ukt-diagnostics-id-portlet/blob/master/src/main/java/life/qbic/AppInfo.java) class. 👍

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,21 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.2.8 (2021-12-20)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:7]: Mb the correct version should also be mentioned in the changelog 👀 
```suggestion
1.2.9 (2021-12-20)
```

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Heya @jenniferboedker. Thanks for adjusting the version in the AppInfo, could you do the same for the Changelog header? I just noticed it sorry 🙈

File: CHANGELOG.rst (hunk 1 above)
Comment (jenniferboedker) [reply CHANGELOG.rst:7]: NO!

Comment (Steffengreiner) [reply CHANGELOG.rst]: 😿

**Review by Steffengreiner — APPROVED** [review]
LGTM 🚀

**Pull request merged**